hschain-utxo-lang/src/Hschain/Utxo/Lang/Sigma/EllipticCurve.hs

```diff
@@ -46,6 +47,8 @@ class ( ByteRepr (ECPoint   a)
   generateScalar    :: IO (ECScalar a)
   fromGenerator     :: ECScalar  a -> ECPoint  a
   fromChallenge     :: Challenge a -> ECScalar a
+  isIdentity        :: ECPoint a -> Bool
```

Comment:
What is `isIdentity`? Implementation for Ed25519 returns True for more that identity element of curve